data-raw/cdc_metadata_creation.R

```diff
@@ -0,0 +1,1323 @@
+cdc_bmiage <- tibble::tribble(
+  ~SEX, ~AGE, ~L, ~M, ~S,
+  1, 24, -2.01118107, 16.57502768, 0.080592465,
```

Comment:
I noticed something odd with the BMI only sheet from CDC where the cells are truncated in the xls for the M column only to be 8 decimal places but when you click on the values themselves they're actually given to 9 decimals, e.g. this one should be `16.575027675`. The weight and height ones are fine and all other columns look good except for this one. Please update so we have an exact match to the values from the CDC spreadsheets

Reply:
ahhh the trick is that there's a difference between the .xls and .csv when you download it

Reply:
wow, i'd guess 99.99% of the time this would never even get spotted then 😄

Reply:
@zdz2101 you're going to hate me 😞 but i just spotted the S column has exactly the same issue! sorry, i thought it was only M!

Reply:
@rossfarrugia It's been updated!